pages/common/paci.md

```diff
@@ -0,0 +1,23 @@
+# paci
+
+> A lightweight and flexible package manager for bash scripts.
+
+- Update the list of available packages and versions (it's recommended to run this before other `paci` commands):
+
+`paci refresh`
+
+- Configure it:
```

Comment:
Define `configure it`?

Reply:
This command will configure paci. It asks where to store temporary data. Where to store the packages itself and you can also enter the url of the main and fallback registry where it will look for packages.
I'm open for suggestions to shortly explain this behaviour, but I thought "Configure it" was sufficient.

Reply:
Perhaps "configure its behaviour"?

Reply:
Ah, I now get the direction. Maybe:
"Setup its behaviour, like where to get packages from and where to store them"

Although it may be a bit too long. ^^

Reply:
Yeah I thought that may be a bit too much. 
Sorry for taking up so much of your valuable time.

But I'm also fine with changing it to "Configure its behaviour" if you two like that better. :)

Reply:
I'm fine with changing it to either :-)

Reply:
Hehe okay I've just added the commit :)